polling_stations/apps/file_uploads/models.py

```diff
@@ -56,7 +56,29 @@ def status(self):
         for f in self.file_set.all():
             if not f.csv_valid:
                 return "Error"
-        return "OK"
+            else:
+                return self.ems_check(f)
```

Comment:
The else isn't needed here.

Reply:
Fixed with https://github.com/DemocracyClub/UK-Polling-Stations/pull/4955/commits/10c545be345bbd157df9c4b8ce7c08db57f08d96